src/glyphs.cpp

```diff
@@ -315,6 +315,8 @@ void RangeAsync(uv_work_t* req) {
                 }
 
                 mutable_fontstack->set_range(std::to_string(baton->start) + "-" + std::to_string(baton->end));
+                mutable_fontstack->set_ascender(ft_face->ascender);
+                mutable_fontstack->set_descender(ft_face->descender);
```

Comment:
Hi, I am currently testing the generated glyphs from this pr in MBGL, I am trying to fix the dis-alignment of texts with mixed fonts (Noto Sans CJK and Arial Unicode MS Regular).  In order to fix it, I need to shift back the `top` value in the metrics in MBGL. Since the `top` value was generated from:
`double top = static_cast<double>(glyph.top) - glyph.ascender;`
In order to do so, I need the `ascender` in the metrics to be 26.6 pixel format instead of unscaled font units.
So I think `ascender` and `descender` are better to be set from `ft_face->size->metrics.ascender` and `ft_face->size->metrics.descender`.

Reply:
👋 @zmiao excited to learn you are digging into this work!

> So I think `ascender` and `descender` are better to be set from `ft_face->size-metrics.ascender` and `ft_face->size->metrics.descender`. 

Feel free to make those modifications to this pull request if that's more accurate!

Reply:
@tristen thanks for the reply, I updated this pr.